Lombiq.VueJs/.npmrc

```diff
@@ -0,0 +1,9 @@
+; See https://pnpm.io/cli/run#shell-emulator for details.
```

Comment:
I had considered refactoring the targets, to accept an array of installation paths instead of just `NodeJsExtensionsInstallationPath`, but that would be a grand change in itself with only a small benefit. Maybe in the future, if/when we find another NE style library project besides Lombiq.VueJs. I don't anticipate we will change this file very often.

Also *LV, as the project name is Lombiq.VueJs. VE does not apply because there is no Extensions there.